Dockerfile.ppc64le.ubi

```
@@ -0,0 +1,275 @@
+###############################################################
+# base stage with basic dependencies
+###############################################################
+FROM registry.access.redhat.com/ubi9/ubi AS base-builder
+
+ARG PYTHON_VERSION=3.12
+ARG OPENBLAS_VERSION=0.3.29
+
+# Set Environment Variables for venv, cargo & openblas
+ENV PATH=/opt/vllm/bin:/root/.cargo/bin:$PATH
```

```suggestion
ENV VIRTUAL_ENV=/opt/vllm
ENV PATH=${VIRTUAL_ENV}/bin:/root/.cargo/bin:$PATH
```
modifying where `/opt/vllm` is hardcoded below